docs/firebase.md

```diff
@@ -20,7 +20,7 @@ of test results.
    * Edit `daq/local/system.conf` to specify the `gcp_cred` setting to point to the downloaded file
      (with a path relative to the `daq/` install directory), e.g.
      `gcp_cred=local/daq-testlab-de56aa4b1e47.json`.
-5. (Re)Start the DAQ install.
+5. (Re)Start DAQ (cmd/run).
```

Comment:
If you want to include cmd/run here then it should be escaped appropirately since it's a command so should appear in the "computer" font -- see above for an example (e.g. daq/local/system.conf)

Reply:
Updated with the proper font.